requirements.txt

```diff
@@ -1,12 +1,3 @@
-joblib==0.17.0
-numpy==1.19.2
-onnx==1.7.0
-onnxconverter-common==1.7.0
-protobuf==3.13.0
-scikit-learn==0.23.2
-scipy==1.5.2
-six==1.15.0
-skl2onnx==1.7.0
-sklearn==0.0
-threadpoolctl==2.1.0
-typing-extensions==3.7.4.3
+scikit-learn==0.24.2
```

Comment:
If you enter `pip freeze` from the command-line it will list all the installed libraries. I generated the original requirements.txt with `pip freeze > requirements.txt`.

Reply:
I went ahead and cleared/deleted the env and run it from scratch. Seems to work just fine with the smaller list.